### ISSUE TITLE
buttons doesn't respect WCAG contrast recommendations

### ISSUE THREAD

**Issue by 532910** [Toxblh/youtube-speed-control#2]
![image](https://user-images.githubusercontent.com/14617699/85131648-df76a580-b23f-11ea-9459-9a5b049423f1.png)

https://webaim.org/resources/contrastchecker/?fcolor=666666&bcolor=333333
https://contrastchecker.com/?c=1081b2&b=17a0db

**Comment by Toxblh** [comment]
Sorry, that is bug of you theme. 
By standard is gray on white
https://webaim.org/resources/contrastchecker/?fcolor=666666&bcolor=333333 which is almost pass 
![image](https://user-images.githubusercontent.com/2198153/85132319-6d4a9480-b230-11ea-93e2-3ce29ef89343.png)

**Comment by 532910** [comment]
No it your bug! You must never define only one color. Both fg and bg must be defined together or not defined at all.

**Comment by 532910** [comment]
Moreover, it's the default dark theme, not a custom one.

**Comment by Toxblh** [comment]
Please send to me a link to the default dark theme.

**Comment by 532910** [comment]
What link are you talking about? It comes with browser.

**Comment by Toxblh** [comment]
So, I use the same, I hope, default dark theme
![image](https://user-images.githubusercontent.com/2198153/85133346-691f7680-b232-11ea-8304-5707581c1da3.png)
But that is doesn't change colors on option page. 
Anyway I will change to more contrast color. But I can't to check how you receive the result of compute colors to dark

**Comment by 532910** [comment]
Try to set `widget.content.allow-gtk-dark-theme` or `browser.display.use_system_colors` with `$GTK_THEME=Adwaita:dark`

The whole ff should be dark:
![image](https://user-images.githubusercontent.com/14617699/85134272-e522ba00-b244-11ea-8185-f7308c06f1ad.png)

**Comment by Toxblh** [comment]
Did it all. Settings of browser and really stay dark. But not option page. 
Anyway. I'll just recreate the option page with more standard approach. Thank you for your time and issue

**Comment by Toxblh** [comment]
Yes found need to reload in new tab
![image](https://user-images.githubusercontent.com/2198153/85135544-26f83400-b236-11ea-9c64-d7d0cfa834da.png)
Got it. Thanks again

**Comment by 532910** [comment]
I confirm it looks fine now, thank you!
